giddy-banana/seeders/20220102102256-demo-articletopviews.cjs

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    /**
+     * Add seed commands here.
+     *
+     * Example:
+     * await queryInterface.bulkInsert('People', [{
+     *   name: 'John Doe',
+     *   isBetaMember: false
+     * }], {});
+    */
+
+    var arttopviews = [];
+    var s2 = '.12.2021';
```

Comment:
That's not exactly last two weeks, is it? :( 

I will fix this up a bit, shall we?